src/widget/wtracktableview.cpp

```diff
@@ -663,6 +667,40 @@ void WTrackTableView::slotOpenInFileBrowser() {
     }
 }
 
+void WTrackTableView::slotRemoveFromDisk() {
+       TrackModel* trackModel = getTrackModel();
+    if (!trackModel) {
+        return;
+    }
+    QMessageBox msgBox (QMessageBox::Information, QObject::tr("Remove From Disk"), QObject::tr("Really delete the files from disk? (Permanent Deletion)"));
```

Comment:
This should be a simple and explicit question, no frills. The text is too complicated for confirming an irreversible action.

Reply:
How about: "Are you sure you want to permanently delete this file?"/"Are you sure you want to permanently delete these files?" It would be nice if the code could distinguish between having a single file selected and multiple files selected to show the appropriate string. Also, I'd like to see the paths of files before deleting them.

Reply:
How about "This will move one or more files to the recycle bin, are you sure you want to continue?"

I feel that some of the less computer literate think of the Recycle Bin as a keyword almost.

Reply:
It is a permanent deletion, from the disk. It doesn't go to the recycle bin, it's simply unlinked? @uklotzde ,  do you have a suggestion as to wording? This is a pretty scary action.